src/layouts/MyLayout.vue

```diff
@@ -127,11 +135,14 @@ export default {
   name: 'MyLayout',
   data() {
     return {
-      leftDrawerOpen: this.$q.platform.is.desktop,
+      rightDrawerOpen: this.$q.platform.is.desktop,
     };
   },
   methods: {
     openURL,
+    logout() {
+      alert('TODO: LOGOUT');
```

Comment:
when linting you'll need this
```suggestion
      // eslint-disable-next-line no-alert
      alert('TODO: LOGOUT');
```